frontend/www/js_templates/api/vrf.js

```diff
@@ -117,9 +117,11 @@ async function deleteVRF(workgroupID, vrfID) {
   try {
     const resp = await fetch(url, {method: 'get', credentials: 'include'});
     const data = await resp.json();
+    if ('error_text' in data) throw(data.error_text);
     console.log(data);
     return data.results;
   } catch(error) {
+    alert('An error occured while deleting a VRF:\n ' + error, ()=> window.location="?action=welcome", {error: true});
```

Comment:
Alerts with a redirect won't work here. Both the welcome and modify_cloud pages use this code.